### ISSUE TITLE
Accelerate launch command fails to run

### ISSUE THREAD

**Issue by hjzhang-forward** [joanrod/star-vector#40]
The following command fails to execute:
accelerate launch --config_file configs/accelerate/deepspeed-8-gpu.yaml starvector/train/train.py config=configs/models/starvector-1b/im2svg-stack.yaml

Error:
train.py

      loss = model(batch)

starvector_arch.py
      def forward(self,  vision_embeds, input_ids, num_generations, attention_mask, num_logits_to_keep)

Here, batch has no input_ids, num_generations, attention_mask, num_logits_to_keep

**Comment by Yukinonooo** [comment]
Have you solved this problem? I also encountered the same problem.

**Comment by Eric-zhang-1111** [comment]
same! Have you solved this problem?
